practice-one/src/services/customerService.js

```diff
@@ -0,0 +1,10 @@
+export const getAllCustomerService = async () => {
+  try {
+    const response = await fetch(
+      "https://64e70a09b0fd9648b78f3e0a.mockapi.io/customers"
```

Comment:
create constant url and constant for path

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/5f6d7efc-720d-46f7-9368-2068847a6638)
![image](https://github.com/datdragon777/React-Training/assets/74806826/682ccd65-167b-4657-b7b4-4459a413d966)